app/app.json

```diff
@@ -19,9 +19,13 @@
       "**/*"
     ],
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.esepeasy.narratemyway",
+      "buildNumber": "1.0.0"
     },
     "android": {
+      "package": "com.esepeasy.narratemyway",
+      "versionCode": 1,
```

Comment:
Should this match the version number?

Reply:
So android has version numbers and version codes, and version codes have to start at 1 and are integers